pytorch_blade/torch_blade/config.py

```diff
@@ -425,6 +426,20 @@ def customize_onnx_opset_version(self, version):
         assert version == _default_onnx_opset_version or version in list(_onnx_stable_opsets) + [_onnx_master_opset]
         self._customize_onnx_opset_version = version
 
+    @property
+    def enable_tf32(self):
+        """The flag to enable TF32.
+
+        :type: bool
+        :default: False
+        """
+        return self._enable_tf32
+
+    @enable_tf32.setter
+    def enable_tf32(self, val):
+        assert isinstance(val, bool), "enable_tf32 should be bool, got {}".format(type(val))
+        self._enable_tf32 = val
+
```

Comment:
I think we can enable TF32 in TensorRT by default. And there is no need to add a new flag like `enable_tf32`. In my opinion, it's confused with too many configurations.